platform-emf/src/test/java/com/softicar/platform/emf/table/configuration/EmfTableConfigurationTest.java

```diff
@@ -0,0 +1,36 @@
+package com.softicar.platform.emf.table.configuration;
+
+import com.softicar.platform.db.runtime.test.AbstractDbTest;
+import com.softicar.platform.emf.test.module.EmfTestModuleInstance;
+import com.softicar.platform.emf.test.simple.EmfTestObject;
+import org.junit.Test;
+
+public class EmfTableConfigurationTest extends AbstractDbTest {
+
+	EmfTableConfiguration<EmfTestObject, Integer, EmfTestModuleInstance> configuration;
+
+	public EmfTableConfigurationTest() {
+
+		configuration = new EmfTableConfiguration<>(EmfTestObject.TABLE);
+	}
+
+	@Test
+	public void testSetBusinessKey() {
+
+		configuration.setBusinessKey(EmfTestObject.UK_NAME_DAY);
```

Comment:
this test method is missing an assert, how about `assertSame(EmfTestObject.UK_NAME_DAY, configuration.get...`

Reply:
it is also redundant to `testGetBusinessKeyWithNonDefaultValue()`